### ISSUE TITLE
`InlineTextEdit` component is not accessible

### ISSUE THREAD

**Issue by ssciolla** [tl-its-umich-edu/canvas-course-manager-next#240]
Issue was reported during the accessibility review on 11/2/21.

"
Issue
Edit title component is not accessible.

Impact
Screen reader and keyboard-only users will not be able to operate it.

Solution
Many possibilities. Possibly the least complicated fix is to make the pencil icon or the parent div (or whatever is responding to onclick) an actual button. Make sure to add an aria-label=""Edit title"" to the control and to pass the focus to the text input when the button is activated. If the element that is listening is the SVG, any of the labelling techniques will do:
https://dequeuniversity.com/rules/axe/4.0/svg-img-alt"
"

**Comment by gsilver** [comment]
Fixed!
